src/seedu/addressbook/ui/TextUi.java

```diff
@@ -126,19 +122,15 @@ public void showResultToUser(CommandResult result) {
         if(resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedback(), DIVIDER);
```

Comment:
adding the divider should be done in the formatter. 
in the current form, you still have some formatting and display in text ui, which is still not SRP, no?